src/models/word/rootWord.ts

```diff
@@ -0,0 +1,34 @@
+
+class RootWord {
+    rootId: number;
+    rootWord: string;
+    meanings: string[];
+
```

Comment:
All models should have optional params being passed in when created (i.e. look at tests using rootwords).